metaflow/plugins/kfp/tests/README.md

```diff
@@ -0,0 +1,31 @@
+# Integration Tests for Metaflow on KFP
+
+These integration tests are based on Metaflow flows specifically designed the catch hard-to-find bugs. They can be run locally, or on Gitlab.
+
+# Local Tests
+
+To run the tests locally, first create your Metaflow config file locally, which can be found at: `/Users/[your_username]/.metaflowconfig/config.json`.
```

Comment:
can you please define what running the tests locally means? because this is running the tests on KFP actually

Reply:
When one looks at https://github.com/zillow/metaflow/blob/9af8a185d191519db093ae657c1d1f4a9688b926/metaflow/plugins/kfp/tests/run_integration_tests.py#L63-L71

It'd be cleaner to not have a `--local` option but to instead have a `--base-image` pytest option.  And when running in CICD to pass in the `--base-image`.

Reply:
@talebzeghmi there is already a `base-image` pytest option, but it's just `image`. See `pytestconfig.getoption('image')`.  The crux of the difference between the local testing and CICD testing is that when testing locally, we include `--no-s3-code-package` because the base image includes the testing flows.